### ISSUE TITLE
Incomplete responses

### ISSUE THREAD

**Issue by runekm** [OptimalBits/redbird#216]
After upgrading from version 0.7.0 to 0.9.0, I started getting incomplete responses from Redbird. This happened when Redbird was proxying responses that had transfer-encoding "chunked". I haven't examined the problem in detail, but it disappeared as soon as I downgraded to version 0.7.0 again.

**Comment by runekm** [comment]
I recently tried upgrading to 0.9.1 again, and the problem reappeared. My ajax requests then starts failing once in a while because just parts of the json that is sent from the server, is received. This seems to only be happening when I send compressed content using http2 - and it only happens in newer versions of Redbird. Disabling compression on the server, seems to fix the problem again. 

I suspect that this may be related to these issues:
https://github.com/OptimalBits/redbird/issues/225
and
https://github.com/OptimalBits/redbird/issues/231

Does anyone else see these problems?

**Comment by davidelbaze** [comment]
Yes I do have the same issues. I had to disable http2 and gzip compression... Unfortunately, redbird does not seems to be upgraded anymore. Maybe should we fork it ?

**Comment by runekm** [comment]
> Yes I do have the same issues. I had to disable http2 and gzip compression... Unfortunately, redbird does not seems to be upgraded anymore. Maybe should we fork it ?

I like Redbird, but it has a lot of small problems. I've spent quite an amount of time trying to find work arounds for some of them. In the end I figured it was much easier to just set up nginx with certbot for letsencrypt certificates. So I did - and I found that this was quite easy to set up, and it has worked flawlessly since.

**Comment by davidelbaze** [comment]
Do you have a link that explains how to setup nginx and certbot ? I mean with automatic renew of certificates, etc. 

Thx for your feedback

**Comment by runekm** [comment]
I used these guides to setup nginx, certbot and http2:
https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04
https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-18-04
https://www.digitalocean.com/community/tutorials/how-to-set-up-nginx-with-http-2-support-on-ubuntu-18-04

**Comment by namipsg** [comment]
same here with React Next.js and express on version 0.10.0 setting httpv2 flag to true will break the webpage load due to corrupt chunk files , downgrading to version 0.7.0 fixes the issue but at the install time i got a warning -> Hey! Lets'Encrypt will STOP WORKING with Greenlock v2 at the end of October
